src/main/java/org/broadinstitute/hellbender/tools/copynumber/PostprocessGermlineCNVCalls.java

```diff
@@ -272,6 +273,29 @@ public void onTraversalStart() {
                 "The SAM sequence dictionary is either not the same for all of the model shards, " +
                         "or is different from the SAM sequence dictionary of calls shards.");
 
+        return sequenceDictionary;
+    }
+
+    /**
+     * Performs various validations on input arguments. Since many of these validations requires loading and parsing
+     * reusable data, we store them as global variables (shard interval lists, sample name, etc).
+     */
+    @Override
+    public void onTraversalStart() {
+        validateArguments();
+
+        numShards = inputUnsortedCallsShardPaths.size();
+
+        /* get intervals from each call and model shard in the provided (potentially arbitrary) order */
+        final List<SimpleIntervalCollection> unsortedIntervalCollectionsFromCalls =
+                getIntervalCollectionsFromPaths(inputUnsortedCallsShardPaths);
```

Comment:
This could potentially be an expensive operation.  It *might* be a good idea to promote these to fields and only initialize them once on demand.  It's more complicated and annoying though so either way 👍.

Reply:
IntelliJ Refactor > Extract Field and > Encapsulate Field FTW!  If I knew how to do the lazy initialization through refactoring this would have been easy peasy.  (Instead it was just easy.)

Reply:
Could you just call these initializing methods in getBestAvailableSequenceDictionary as well?  (Matilda is preventing me from doing a detailed review, so please ignore if I'm wrong!)